apps/website/src/pages/hackathon/submit.astro

```diff
@@ -0,0 +1,402 @@
+												promotion from being conducted as planned.
+											</p>
+
+											<p>
+												By submitting your email address to CodeTV as part of
+												this promotion, you agree and consent that you may be
+												contacted by CodeTV or one of CodeTV's successors or
+												assigns to notify you if you are selected as a winner,
+												send you updates about future CodeTV promotions, and
+												send you other CodeTV news. You may opt out of any
+												emails from CodeTV by clicking the unsubscribe button at
+												the bottom of any email received from CodeTV.
+												Additionally, CodeTV will share your name and email
+												address with the sponsor of this promotion. By
+												submitting your name and email address to CodeTV as part
+												of this promotion, you agree and consent to allow CodeTV
+												to share your name and email address with the sponsor of
+												this promotion. Once shared, CodeTV will have no control
+												over how the sponsor communicates with you or what the
+												sponsor does with your name and email address; you must
+												communicate directly with the sponsor to manage any
+												communications with the sponsor.
+											</p>
+
+											<p>
+												You further agree that, unless you affirmatively opt out
+												by unchecking the box in the entry form, the sponsor of
+												this promotion may display your App at its discretion as
+												an example of how to use their tool or technology to
+												build an app.
+											</p>
+										</div>
+
+										<div class="checkbox-group">
+											<input
+												type="checkbox"
+												name="agreeTerms"
+												id="agreeTerms"
+												class="input checkbox"
+												required
+											/>
+											<label for="agreeTerms">
+												I agree to the terms and conditions
+											</label>
+										</div>
+									</div>
+
+									<div class="field">
+										<p class="opt-out-label">
+											Only check this box if you DO NOT want your app to be
+											shared by the sponsor
+										</p>
+
+										<div class="checkbox-group">
+											<input
+												type="checkbox"
+												name="optOutSponsorship"
+												id="optOutSponsorship"
+												class="input checkbox"
+											/>
+											<label for="optOutSponsorship">
+												I DO NOT want the sponsor to showcase my web app as an
+												example of using their tool
+											</label>
+										</div>
+									</div>
+								</section>
+
+								<section class="form-section">
+									<button type="submit" class="button submit-btn">
+										Submit
+									</button>
+								</section>
+							</form>
+						</>
+					)
+				}
+			</section>
+		</SignedIn>
+
+		<SignedOut>
+			<SignIn forceRedirectUrl={url.toString()} />
+		</SignedOut>
+	</main>
+</Layout>
+
+<style>
```

Comment:
is this CSS duplicated from the other form? if so, can we open an issue to track it so I can go back and dedupe styles later?